client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -1,152 +1,217 @@
+import { formatNumberMetric } from 'calypso/lib/format-number-compact';
+import version_compare from 'calypso/lib/version-compare';
 import PluginIcon from 'calypso/my-sites/plugins/plugin-icon/plugin-icon';
+import PluginRatings from 'calypso/my-sites/plugins/plugin-ratings/';
 import { siteObjectsToSiteIds } from 'calypso/my-sites/plugins/utils';
 import { getSitesWithPlugin } from 'calypso/state/plugins/installed/selectors';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
-import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
 import { PluginsBrowserElementVariant } from './types';
 
 import './style.scss';
 
 const PREINSTALLED_PLUGINS = [ 'Jetpack by WordPress.com', 'Akismet', 'VaultPress' ];
 
-class PluginsBrowserListElement extends Component {
-	static defaultProps = {
-		iconSize: 40,
-		variant: PluginsBrowserElementVariant.Compact,
-	};
-
-	getPluginLink() {
-		if ( this.props.plugin.link ) {
-			return this.props.plugin.link;
+const PluginsBrowserListElement = ( props ) => {
+	const {
+		isPlaceholder,
+		site,
+		plugin = {},
+		iconSize = 40,
+		variant = PluginsBrowserElementVariant.Compact,
+		currentSites,
+	} = props;
+
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+
+	const selectedSite = useSelector( getSelectedSite );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, selectedSite?.id ) );
+	const sitesWithPlugin = useSelector( ( state ) =>
+		isJetpack && currentSites
+			? getSitesWithPlugin( state, siteObjectsToSiteIds( currentSites ), plugin.slug )
+			: []
+	);
+
+	const dateFromNow = useMemo(
+		() =>
+			plugin.last_updated ? moment.utc( plugin.last_updated, 'YYYY-MM-DD hh:mma' ).fromNow() : null,
+		[ plugin.last_updated ]
+	);
+
+	const pluginLink = useMemo( () => {
+		if ( plugin.link ) {
+			return plugin.link;
 		}
 
-		let url = '/plugins/' + this.props.plugin.slug;
-		if ( this.props.site ) {
-			url += '/' + this.props.site;
+		let url = '/plugins/' + plugin.slug;
+		if ( site ) {
+			url += '/' + site;
 		}
 		return url;
-	}
+	}, [ plugin, site ] );
 
-	trackPluginLinkClick = () => {
+	const trackPluginLinkClick = useCallback( () => {
 		recordTracksEvent( 'calypso_plugin_browser_item_click', {
-			site: this.props.site,
-			plugin: this.props.plugin.slug,
-			list_name: this.props.listName,
+			site: site,
+			plugin: plugin.slug,
+			list_name: props.listName,
 		} );
-	};
+	}, [ site, plugin, props.listName ] );
 
-	isWpcomPreinstalled() {
-		if ( this.props.plugin.isPreinstalled ) {
+	const isWpcomPreinstalled = useMemo( () => {
+		if ( plugin.isPreinstalled ) {
 			return true;
 		}
 
-		if ( ! this.props.site ) {
+		if ( ! site ) {
 			return false;
 		}
 
-		return ! this.props.isJetpackSite && includes( PREINSTALLED_PLUGINS, this.props.plugin.name );
-	}
+		return ! isJetpack && includes( PREINSTALLED_PLUGINS, plugin.name );
```

Comment:
Since we want to refactor away loadash, would `PREINSTALLED_PLUGINS.include( plugin.name )` work here?

Reply:
done 8284966f3a808426e9a78433dd75c7ee74df16d7